src/Components/DurationLineChart.tsx

```diff
@@ -112,7 +116,13 @@ const DurationLineChart: React.FC<DurationLineChartProps> = ({
 }): ReactElement => {
   //if there's no data return nothing
   if (!data || data.length <= 0) {
-    return <React.Fragment></React.Fragment>
+    return !data ? (
+      <React.Fragment></React.Fragment>
+    ) : (
```

Comment:
`!!data && <React.Fragment><p>No completed runs</p></React.Fragment>`?
That seems backwards to me but is the same logic as you have currently.

Reply:
I was trying to do different things based on if data was null vs an empty array.  Simplified logic to make it more clear.